freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientException.java

```diff
@@ -1,19 +1,25 @@
 package com.sequenceiq.freeipa.client;
 
 public class FreeIpaClientException extends Exception {
+
+    private final int statusCode;
```

Comment:
I think the statusCode should be `OptionalInt` since its not always present.

Reply:
changed